zxinglight/_zxinglight.cpp

```diff
@@ -105,7 +111,7 @@ static PyObject* zxing_read_codes(PyObject *self, PyObject *args) {
     PyBytes_AsStringAndSize(python_image, &image, &image_size);
 
     vector<string> *results = _zxing_read_codes(
-        image, image_size, width, height, barcode_type, try_harder, hybrid
+        image, image_size, width, height, barcode_type, try_harder, hybrid, multi
```

Comment:
Please, make sure all your C++ code is formatted the same way as the existing code. Usually, I use uncrustify with default config, 4 spaces instead of tabs for indentation and maximum line length of 100 characters. I'm sorry for missing linting on CI.

Reply:
No worries. Your code is a pleasure to work with anyhow.
I've gone through my changes and I only found one line that was over 100 chars, but I've fixed it up now.